### PR TITLE
Updated strings for `LINE Messenger` as changed in their developer portal

### LINKED ISSUE #4526: LINE messenger has changed their developer portal, making some helptexts outdated
### ⚠️ Please verify that this question has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

I'm trying to add LINE messenger alerts into Uptime kuma but everytime I try to run a test, I get 401'ed with this error:
image

LINE's dev portal doesn't have have a Channel access token anymore but instead seems to have Channel secrets instead, does this change anything on how Uptime Kuma communicates with LINE's API?

### 📝 Error Message(s) or Log

_No response_

### 🐻 Uptime-Kuma Version

1.23.11

### 💻 Operating System and Arch

UNRAID 6.12.8 - x64 (Intel)

### 🌐 Browser

Brave Version 1.63.162 Chromium: 122.0.6261.69 (Official Build) (64-bit)

### 🖥️ Deployment Environment

- Runtime: Docker (UNRAID 6.12.8)
- Database: Unknown (whatever is default)
- Filesystem used to store the database on: btrfs (in Docker)
- number of monitors: 12

### PR DESCRIPTION
**Pull request by Vista2003**
⚠️⚠️⚠️ Since we do not accept all types of pull requests and do not want to waste your time. Please be sure that you have read pull request rules:
https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma

Tick the checkbox if you understand [x]:
- [x] I have read and understand the pull request rules.

# Description

Fixes #4526 

This is just some string changes to reflect how the LINE dev environment looks now

## Type of change

Please delete any options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- User interface (UI)

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I ran ESLint and other linters for modified files
- [x] I have performed a self-review of my own code and tested it
- [ N/A] I have commented my code, particularly in hard-to-understand areas (including JSDoc for methods)
- [x] My changes generates no new warnings
- [ ] My code needed automated testing. I have added them (this is optional task)

## Screenshots (if any)

Please do not use any external image service. Instead, just paste in or drag and drop the image here, and it will be uploaded automatically.

### CODE REVIEW

File: src/components/notifications/Line.vue [hunk 1]
```diff
@@ -4,14 +4,14 @@
         <HiddenInput id="line-channel-access-token" v-model="$parent.notification.lineChannelAccessToken" :required="true" autocomplete="new-password"></HiddenInput>
```
[the remark this answers is not in the document]
Comment (Vista2003) [reply src/components/notifications/Line.vue:4]: Yep, you're right. I've changed that text now

File: src/lang/en.json [hunk 2]
```diff
@@ -532,10 +532,12 @@
     "Recipients": "Recipients",
     "Access Token": "Access Token",
     "Channel access token": "Channel access token",
+    "ChannelAccessTokenLongLived": "Channel access token (Long-lived)",
```
Comment (CommanderStorm) [inline src/lang/en.json:535]:
```suggestion
    "Channel access token (Long-lived)": "Channel access token (Long-lived)",
```

File: src/components/notifications/Line.vue [hunk 3]
```diff
@@ -1,17 +1,17 @@
 <template>
     <div class="mb-3">
-        <label for="line-channel-access-token" class="form-label">{{ $t("Channel access token") }}</label>
+        <label for="line-channel-access-token" class="form-label">{{ $t("ChannelAccessTokenLongLived") }}</label>
```
Comment (CommanderStorm) [inline src/components/notifications/Line.vue:3]:
```suggestion
        <label for="line-channel-access-token" class="form-label">{{ $t("Channel access token (Long-lived)") }}</label>
```

**Review by CommanderStorm — APPROVED** [review]
We prefer to keep the english translation and the string as close as possible if possible ^^

**Review by Vista2003 — COMMENTED** [review]
Understandable, thanks for the edit.

**Comment by CommanderStorm** [comment]
All changes in this PR are small and uncontroversial.
⇒ merging with junior maintainer approval as by contribution guide

**Pull request merged**